.l10nignore

```diff
@@ -0,0 +1 @@
+js/firstrunwizard.js
```

Comment:
This should also contain `js/firstrunwizard.js.map`

Reply:
Not really needed, because the l10n bot only checks for PHP, HTML and JS files ;)